common/djangoapps/student/tests/test_refunds.py

```diff
@@ -206,9 +208,18 @@ def test_multiple_refunds_dashbaord_page_error(self):
                 enrollment=self.enrollment,
                 namespace='order',
                 name='order_number',
-                value=order_number
+                value=self.ORDER_NUMBER
             ))
 
-        self.client.login(username="jack", password="test")
+        self.client.login(username=self.user.username, password=self.USER_PASSWORD)
         resp = self.client.post(reverse('student.views.dashboard', args=[]))
         self.assertEqual(resp.status_code, 200)
+
+    def test_refund_cutoff_date_with_client_error(self):
+        """ Verify that dashboard will not throw internal server error if HttpClientError
+         raised while getting order detail for ecommerce.
+         """
+        self.client.login(username=self.user.username, password=self.USER_PASSWORD)
+        with mock_order_endpoint(order_number=self.ORDER_NUMBER, exception=HttpClientError):
+            response = self.client.post(reverse('student.views.dashboard', args=[]))
```

Comment:
@ahsan-ul-haq  I am mocking exception which is not actually logging the message. So I can't test log message here.